simtool.h

```diff
@@ -22,6 +22,9 @@
 
 #include "display/viewport.h"
 
+#include "gui/simwin.h"
+#include "gui/city_info.h"
```

Comment:
Should these be in the `.cc` file rather than the header file?

Reply:
noticed that those two were already in the `.cc` so I've removed them in the header file.